ruby/README.md

```diff
@@ -51,6 +52,17 @@ puts MyTestMessage.encode_json(mymessage)
 Installation from Source (Building Gem)
 ---------------------------------------
 
+Protocol Buffers uses the [ffi](https://github.com/ffi/ffi) gem to provide a
```

Comment:
Could we instead say "Protocol Buffers has an experimental backend which uses the ffi gem to provide a unified C-based implementation..."

"Protocol Buffers uses..." makes it sound like FFI is the default.

Reply:
Updated.